saltgui/static/scripts/router.js

```diff
@@ -92,5 +92,5 @@ class Router {
 
 }
 
-window.addEventListener('load', new Router());
+window.addEventListener('load', () => new Router());
```

Comment:
I spend at least an hour on that too. decided it was not caused by SaltGui. guess I was wrong...